spec/acceptance/propagation/prop_file_spec.rb

```diff
@@ -47,25 +46,23 @@ def acl_manifest(target_name, file_content, rights, prop_type, affects_child_typ
 
     verify_content_command = "cat /cygdrive/c/temp/#{target_name}"
 
-# 4c734680aca3b3781ae9fb211759a5610c6679a8 changed how permissions are emitted
-# during a `puppet agent` / `puppet apply` (but not `puppet resource`), so that
-# instead of emitting a [Puppet::Type::Acl::Ace] for rendering to the console
-# a [Hash] is emitted in the permissions_to_s method
-# Puppet 4 and 5 have different behavior for rendering this data structure
-    verify_manifest_pup4 = /\{ affects => 'self_only', identity => '.*\\bob', rights => \['full'\s+\] \}/
-    verify_manifest_pup5 = /\{"identity"=>".*\\bob", "rights"=>\["full"\], "affects"=>:self_only\}/
+    # 4c734680aca3b3781ae9fb211759a5610c6679a8 changed how permissions are emitted
+    # during a `puppet agent` / `puppet apply` (but not `puppet resource`), so that
+    # instead of emitting a [Puppet::Type::Acl::Ace] for rendering to the console
+    # a [Hash] is emitted in the permissions_to_s method
+    # Puppet 4 and 5 have different behavior for rendering this data structure
+    verify_manifest_pup4 = %r{\{ affects => 'self_only', identity => '.*\\bob', rights => \['full'\s+\] \}}
```

Comment:
I thought the point of using "%r{}" notation was so that you don't have to use escape characters for things like brackets, so I think this will break in acceptance unless we remove the escape characters.